### PR TITLE
add  BADSHAH-AMAN

### LINKED ISSUE #685: Aman Upadhyay
image

### PR DESCRIPTION
**Pull request by BADSHAH-AMAN**
resolve #685

Aman Upadhyay

### CODE REVIEW

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]
@BADSHAH-AMAN please place your name at the end of the file and then we will see to merging (:

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]
Also, please change your issue number to 685

**Review by BADSHAH-AMAN — COMMENTED** [review]
I have changed it the way you told me

**Review by BADSHAH-AMAN — COMMENTED** [review]
Done Thanks for the suggestion Actually I am new to this

**Review by AvidCoder101 — APPROVED** [review]
LGTM, @schmelto I think that this PR can be merged now? Thanks for contributing to this repo 🚀

File: data.json [hunk 1]
```diff
@@ -1300,5 +1300,11 @@
     "githubUsername": "Mo-Shakib",
     "imageUrl": "https://user-images.githubusercontent.com/50780268/131604251-f07f6811-d795-448f-a7b8-0f13b58c602f.png",
     "issueNumber": 675
+  },
+   {
+    "name": "Aman Upadhyay",
+    "githubUsername": "BADSHAH-AMAN",
+    "imageUrl": "https://user-images.githubusercontent.com/76415079/132032053-210481b4-882d-4314-875d-fbd4695142da.png",
```
Comment (Panquesito7) [inline data.json:1307]: I would recommend more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/132057221-2e7c6dc4-a664-4565-9909-101b8fadfa4b.png",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**